sr_robot_commander/test/test_robot_commander.py

```diff
@@ -0,0 +1,652 @@
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+from __future__ import absolute_import
+
+import collections
+from builtins import round
+import copy
+import rospy
+import rostest
+from unittest import TestCase
+from sr_robot_commander.sr_robot_commander import SrRobotCommander, MoveGroupCommander, PlanningSceneInterface
+from geometry_msgs.msg import Pose, PoseStamped
+from control_msgs.msg import FollowJointTrajectoryActionGoal
+from moveit_msgs.msg import RobotState, RobotTrajectory
+from moveit_msgs.srv import GetPositionFK, SaveRobotStateToWarehouse
+from trajectory_msgs.msg import JointTrajectoryPoint, JointTrajectory
+from std_msgs.msg import Header
+from sensor_msgs.msg import JointState
+from moveit_commander.exception import MoveItCommanderException
+from moveit_commander import conversions
+from actionlib_msgs.msg import GoalStatusArray
+import tf2_ros
+import time
+
+# Some of the test cases are consisting of 1 line. In case of these methods the test verifies if
+# the API of moveit_commander changed - i.e. change of methods name, number of arguments, return type
+
+PKG = "sr_robot_commander"
+CONST_RA_HOME_ANGLES = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
+                        'ra_shoulder_lift_joint': -1.57, 'ra_wrist_1_joint': -0.73,
+                        'ra_wrist_2_joint': 1.57, 'ra_wrist_3_joint': 0.00}
+
+CONST_EXAMPLE_TARGET = {'ra_shoulder_pan_joint': 0.2, 'ra_elbow_joint': 1.80,
+                        'ra_shoulder_lift_joint': -1.37, 'ra_wrist_1_joint': -0.52,
+                        'ra_wrist_2_joint': 1.57, 'ra_wrist_3_joint': 0.00}
+
+
+class TestSrRobotCommander(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        rospy.wait_for_message("/move_group/status", GoalStatusArray)
+        cls.robot_commander = SrRobotCommander("right_arm")
+        cls.robot_commander.set_planner_id("RRTstarkConfigDefault")
+        cls.robot_commander.set_planning_time(3)
+        cls.eef = cls.robot_commander.get_end_effector_link()
+
+    def reset_to_home(self):
+        self.robot_commander._reset_plan()
+        plan = self.robot_commander.plan_to_joint_value_target(CONST_RA_HOME_ANGLES, angle_degrees=False,
+                                                               custom_start_state=None)
+        self.robot_commander.execute_plan(plan)
+
+    def compare_poses(self, pose1, pose2, tolerance=0.01):
+        p1_list = [pose1.position.x, pose1.position.y, pose1.position.z,
```

Comment:
change names to pose1_list and pose2_list maybe?

Reply:
Done